sycl/test/esimd/intrins_trans.cpp

```diff
@@ -45,14 +45,14 @@ SYCL_ESIMD_FUNCTION SYCL_EXTERNAL simd<float, 16> foo() {
   simd<ushort, VL> pred;
   v_addr += offsets;
 
-  __esimd_flat_atomic0<EsimdAtomicOpType::ATOMIC_INC, uint32_t, VL>(
+  __esimd_flat_atomic0<EsimdAtomicOpType::inc, uint32_t, VL>(
```

Comment:
Since you are editing this anyway, this needs 1 more step (`EsimdAtomicOpType` itself is deprecated)
```suggestion
  __esimd_flat_atomic0<atomic_op::inc, uint32_t, VL>(
```

Reply:
> `EsimdAtomicOpType` itself is deprecated

The compiler gives a message only about using `ATOMIC_*`.
I suggest we update the tests together with adding corresponding diagnostics.